internal/namespaces/k8s/v1beta4/custom_node.go

```diff
@@ -0,0 +1,57 @@
+package k8s
+
+import (
+	"context"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-cli/internal/human"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	nodeActionTimeout = 10 * time.Minute
+)
+
+//
+// Marshalers
+//
+
+// nodeStatusMarshalerFunc marshals a k8s.ClusterStatus.
+var (
+	nodeStatusAttributes = human.Attributes{
+		k8s.NodeStatusCreating:      color.FgBlue,
```

Comment:
isn't this list of attributes similar to one used in another command ? 

can't we use a common list, to avoid code duplication ?

Reply:
Not really, there are some for cluster, pool and nodesm which are not the same